docs/building-apps/moneygram-access-integration-guide.mdx

```diff
@@ -185,10 +216,27 @@ You will need the following pieces of information:
 - The amount the user would like to withdraw / cash-out
   - This should be collected from the user prior to initiating this transaction
 
-The following code can be used as a reference for implementing this logic yourself. This code is not necessarily production-ready.
+The following code can be used as a reference for implementing this logic yourself.
 
 <CodeExample>
 
+```ts
+// Use same "anchor" object from previous step.
+const { url, id } = await anchor.sep24().withdraw({
+  authToken: authToken, // Use same "authToken" string from previous step
+  withdrawalAccount: FUNDS_STELLAR_KEYPAIR.public_key,
+  assetCode: ASSET_CODE, // Should be "USDC"
+  lang: "en",
+  extraFields: {
+    amount: "<amount to withdraw / cash-out>",
```

Comment:
Should note that this field is optional and only mandatory for custodial wallets